### PR TITLE
Silver Dagger and Silver Psycross effects Reworked, Witch Hunters get Silver Daggers

### PR DESCRIPTION
**Pull request by wgtjunior743**
## About The Pull Request

- Fixed the silver dagger and psycross effects on vampires , werewolves are not merged when this PR was made

- Silver Dagger will burn and stun lesser spawn and also reveal their form on hit, the vampire lord won't get burned and will be stunned, if the vampire lord has reached lvl 4 and someone attack him with the silver dagger ,the person attacking will get stunned, the silver dagger on lessers trigger a new debuff called "Silver Curse", for 5 seconds vampires won't be able to regen, the silver curse trigger has a 10 seconds cd on the dagger so you can't spam it.

- Silver psycross on the neck or bracers make you immune to transfix, if you roll high enough like the game already does with arcanic or holy you can notice who did magic on you and it will write as "blood magic" and not "unholy magic" since you will know more about who did it.

- Vampires can't pick up or store the silver dagger nor silver psycross, they will burn and get stunned, the lord vampire will only get stunned unless he gets lvl 4 , then he will be able to pick the silver daggers or silver psycross without anything happening.

- As soon as the Werewolves are merged, i will work on making the silver dagger effect on them, ofcourse not like the vampires.

- ISSUE, silver dagger  or any weapon may dissapear because once it stuck on the vampire and the vampire becomes dust ,it is deleted, has a higher chance of happening with the silver dagger since you may burn the vampire hit him multiple times and stuck your dagger making it be deleted once the vampire becomes dust(happened many times in testing). (bug)

## Why It's Good For The Game

Give the witch hunters / inquisitors tools against vampires, making them a real threat for the vampires, you can't smelt silver gobblets or anything at the moment to make silver, so in total 4 daggers are possible 3 from the inquisitor and witch hunters and one from the Lblacksmith, also 4  silver psycrosses and 1 extra at the church and 3 from the inquisitor and witch hunters, in total these are rare items that are very scarce but at the same powerful enough against vampires.
Let the Vampire Hunt begin.

### CODE REVIEW

**Comment by GreenHoodiedEthan** [comment]
Will elvish sabers get the same treatment due to their status as silver weapons, or just the dagger?

**Comment by wgtjunior743** [comment]
> Will elvish sabers get the same treatment due to their status as silver weapons, or just the dagger?

I think they are silver for a place holder since we can't really get It, silver will be a really rare or or a very expensive import, for the sake of balancing for now only daggers are like that, maybe in the future the elvish sabers will also be

File: code/game/objects/items/rogueweapons/melee/knives.dm [hunk 1]
```diff
@@ -178,30 +178,87 @@
...
+/obj/item/rogueweapon/huntingknife/idagger/silver/funny_attack_effects(mob/living/target, mob/living/user = usr, nodmg)
+	if(world.time < src.last_used + 100)
+		to_chat(user, "<span class='notice'>The silver effect is on cooldown.</span>")
+		return
+	
+	
+	
 	. = ..()
 	if(ishuman(target))
+		world.log << "Reached ishumantarget"
```
Comment (JTGSZ) [inline code/game/objects/items/rogueweapons/melee/knives.dm:226]: Left a debug message in

Comment (wgtjunior743) [reply code/game/objects/items/rogueweapons/melee/knives.dm:226]: oh true, thanks

**Pull request merged**